client/src/utils/fuzzyDateOffset.js

```diff
@@ -1,4 +1,12 @@
 export const fuzzyDateOffset = (datestring) => {
+    if (!datestring) return "Never Updated";
```

Comment:
Is this possible? I thought userUpdatedAt defaulted to the current time on record creation... are there records that are actually null?

Reply:
There were records in my local database, which is why it made its way into the PR. I can remove it but I don't see it causing an issue - if we see it in a deployment then that might indicate another issue but I don't think we will.